GOVERNANCE.md

```diff
@@ -9,17 +9,11 @@ An up to date list of the maintainers can be found at the [Eclipse project page]
 
 ## Bi-weekly meetup
 
-Bi-weekly every first and third Thursday for 1 hour. Everyone is welcome to join.
+The developer meetup happens bi-weekly every first and third Thursday from 17:00 - 18:00 CET. Everyone is welcome to join.
```

Comment:
I'd remove `bi-weekly` and only mention 1st and 3rd. Take this April/May as an example. There's a dev meeting April 1, April 15 and next is May 6? Am I correct?
What about `The developer meetup is held every first and third Thursday each month from 17:00 - 18:00 CET`

Reply:
Personally I would prefer the bi-weekly since it's easier for me to remember that there was no meeting last week and therefore this week must be one instead of looking at the calendar and count til three ... but as long as you don't decide for the forth Thursday, that also works for me